### PR TITLE
Fix for various font issues

### PR DESCRIPTION
**Pull request by peterbrittain**
This change should:

- Fix any fonts that use unicode line separators (incluting various tlf files)
- Address the smushing issue for crawford2
- Limit tests for tlf files (where we clearly match figet line splitting logic and not toilet)

TBH, I think pyfiglet does the right thing and so we shouldn't force the dodgy line splitting in toilet.

Basically addresses most of the issues in #116.

### CODE REVIEW

File: pyfiglet/__init__.py [hunk 1]
```diff
@@ -336,10 +340,11 @@ def __char(data):
             # Load German Umlaute - the follow directly after standard character 127
             if data:
```
Comment (stefanor) [inline pyfiglet/__init__.py:341]: Maybe drop this `if` then?

Comment (peterbrittain) [reply pyfiglet/__init__.py:341]: Oops.  That was an attempt to fix gradient, which is actually a broken file...  Some definitions are missing lines and so pyfiglet runs out of data on the last character.  I originally thought it was missing the enter definition, but it dies part-way through in `__char()`.

The real question should probably be whether pyfiglet should soldier on, or reject the font entirely.

Comment (stefanor) [reply pyfiglet/__init__.py:341]: My assumption, when I added that was that fonts would either have all these german umlaut extensions or not. Just a wild guess, I don't know how the fonts are usually structured.

Comment (peterbrittain) [reply pyfiglet/__init__.py:341]: Yeah - that's what figlet does too, but it's clearly happy to stop processing mid character and just use what it found.

Comment (pwaller) [reply pyfiglet/__init__.py:341]: What's the conclusion here. Ready to go or is something outstanding here? I don't have a strong opinion, though possibly it's better to follow figlet's behaviour?

Comment (peterbrittain) [reply pyfiglet/__init__.py:341]: I'll keep it simple and just remove the erroneous check.

**Comment by pwaller** [comment]
(I hope you don't mind, I'm attempting to enable the actions so we can see the tests pass. Not quite sure why they aren't running already. Are you happy for me to merge assuming that they do?)

**Comment by pwaller** [comment]
There we go. Tests are running now.

**Pull request merged**